kerchunk/combine.py

```diff
@@ -107,6 +107,7 @@ def __init__(
         preprocess=None,
         postprocess=None,
         out=None,
+        append=False,
```

Comment:
@martindurant Missing a :param append: entry in the docstring ?

Reply:
Yep, I had intended this to be just the start of a bigger PR, but since it's useful already, I'll tidy it up and remove this for a separate PR